detect_variants/vep.cwl

```diff
@@ -25,6 +25,8 @@ inputs:
         inputBinding:
             prefix: "-i"
             position: 1
+    reference:
+        type: File
```

Comment:
Is the reference only required when `--hgvs` is specified?  If so, should this be optional (or possibly a [dependent parameter](http://www.commonwl.org/v1.0/UserGuide.html#Record_inputs,_dependent_and_mutually_exclusive_parameters))?